scoop/src/main/java/com/lyft/scoop/Router.java

```diff
@@ -111,20 +120,21 @@ public void onCreate(Scoop root) {
         this.root = root;
     }
 
-    public void onCreate(Scoop root, Screen defaultScreen) {
-        this.root = root;
-
-        if (backStack.isEmpty()) {
-            backStack.push(screenScooper.createScreenScoop(defaultScreen, root));
-        }
-
-        performScoopChange(backStack.peek(), defaultScreen, null, TransitionDirection.ENTER);
-    }
-
     public boolean hasActiveScreen() {
         return !backStack.isEmpty();
     }
 
+
+    private boolean isBackstackEmpty(final Screen screen) {
```

Comment:
since it is not just boolean, but a method that actually modify state. name it checkEmptyBackstack